indexing.sql

```diff
@@ -0,0 +1,59 @@
+CREATE USER indexed_cars_user;
+CREATE DATABASE indexed_cars;
+
+--49 ms no index
+select distinct make_title
+from car_models
+where make_code = 'LAM';
+
+--34 ms no index
+select distinct model_title
+from car_models
+where make_code = 'NISSAN'
+and model_code = 'GT-R';
+
+--32 ms no index
+select make_code, model_code, model_title, year
+from car_models
+where make_code = 'LAM';
+
+--309 ms no index
+select *
+from car_models
+where year between 2010 and 2015;
+
+--91 ms no index
+select *
+from car_models
+where year = 2010;
+
+CREATE INDEX IX
+ON car_models (make_code, model_code);
```

Comment:
Although this is fine for this exercise. It's best to make two different indexes, one for make_code and another for model_code. The reason being, what you use in your `WHERE` is what should make up your indexes. There are no queries that use both make_code and model_code in the same `WHERE`